doc/api/replication.md

```diff
@@ -0,0 +1,106 @@
+# Thali Cordova Peer Communication API #
+
```

Comment:
1. Renamed to `connectivity.md` to deal with connectivity between peers
2. Changed to the following:
This is the interface to be implemented by the native layer for handling local discovery and peer to peer communication.  The [`ThaliEmmitter`](thaliemitter.md) class is for use by developers which communicates with the Thali Peer Communication API.